### PR TITLE
🎨 Switched to eslint plugin

### PR DESCRIPTION
**Pull request by AileenCGN**
no issue

Use our standardised eslint plugin and fix linting issues.

The commits can me squashed. I only left them separate for readability reasons.

### CODE REVIEW

File: shipitfile.js [hunk 1]
```diff
@@ -6,14 +6,14 @@ function init(shipit) {
             yarn: true,
             workspace: './',
             deployTo: '/opt/gscan/',
-            ignores: ['.git', '.gitkeep', '.gitignore', '.jshintrc', 'node_modules'],
+            ignores: ['.git', '.gitkeep', '.gitignore', '.jshintrc', 'node_modules']
```
Comment (kirrg001) [inline shipitfile.js]: > '.jshintrc'

Comment (AileenCGN) [reply shipitfile.js]: 👍 Fixed and pushed!

File: app/index.js [hunk 2]
```diff
@@ -76,9 +75,9 @@ app.use(function (req, res, next) {
     next(new errors.NotFoundError({message: 'Page not found'}));
 });
 
-app.use(function (err, req, res, next) {
+app.use(function (err, req, res) {
```
Comment (kirrg001) [inline app/index.js]: This doesn't work anymore.
See for example: https://github.com/expressjs/express/issues/2896

File: lib/read-theme.js [hunk 3]
```diff
@@ -10,15 +10,17 @@ var Promise = require('bluebird'),
...
 readThemeStructure = function readThemeFiles(themePath, subPath, arr) {
     themePath = path.join(themePath, '.');
     subPath = subPath || '';
 
     var tmpPath = os.tmpdir(),
-        inTmp = themePath.substr(0, tmpPath.length) === tmpPath,
-        arr = arr || [];
+        inTmp = themePath.substr(0, tmpPath.length) === tmpPath;
+
+    arr = arr || [];
```
Comment (kirrg001) [inline lib/read-theme.js]: where is the variable definition

File: shipitfile.js [hunk 4]
```diff
@@ -6,14 +6,14 @@ function init(shipit) {
             yarn: true,
             workspace: './',
             deployTo: '/opt/gscan/',
-            ignores: ['.git', '.gitkeep', '.gitignore', '.jshintrc', 'node_modules'],
+            ignores: ['.git', '.gitkeep', '.gitignore', 'node_modules']
```
Comment (kirrg001) [inline shipitfile.js]: you need to add the eslint files?

File: lib/read-theme.js (hunk 3 above)
Comment (AileenCGN) [reply lib/read-theme.js]: It's passed to the function?

File: shipitfile.js (hunk 4 above)
Comment (AileenCGN) [reply shipitfile.js]: 🤦‍♀️

File: lib/read-theme.js (hunk 3 above)
Comment (kirrg001) [reply lib/read-theme.js]: ok i see 👍

File: app/index.js (hunk 2 above)
Comment (AileenCGN) [reply app/index.js]: Good call! Didn't know that!

**Comment by AileenCGN** [comment]
@kirrg001 This should be good to go now

**Pull request merged**